functions/go/upsert-resource/README.md

```diff
@@ -0,0 +1,128 @@
+# upsert-resource
+
+### Overview
+
+<!--mdtogo:Short-->
+
+Upsert a resource to the input list of resources.
+
+<!--mdtogo-->
+
+### Synopsis
+
+<!--mdtogo:Long-->
+
+Upsert is an operation that adds a resource(uniquely identified by Group, Kind, Name and Namespace(GKNN))
```

Comment:
nit: spacing and parentheses inside parentheses.

Reply:
Updated.